_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -187,27 +187,27 @@ The following parameters allow you to scan S3 objects. All options can be config
+`buckets` | Yes  | List | A list of [bucket](#bucket) to scan.
 `scheduling` | No  | List | The configuration for scheduling periodic scans on all buckets. `start_time`, `end_time` and `range` can not be used if scheduling is configured. 
 
-### bucket
+### bucket<a name="bucket"></a>
 
 Option | Required | Type | Description
-:--- | :--- | :--- | :---
-`bucket` | Yes | String list | Provides options for each bucket.
+:--- | :--- |:-----| :---
+`bucket` | Yes | Map  | Provides options for each bucket.
 
-You can configure the following options inside the bucket setting.
+You can configure the following options inside the [bucket](#bucket) setting.
 
 Option | Required | Type | Description
 :--- | :--- | :--- | :---
 `name` | Yes | String   | The string representing the S3 bucket name to scan.
 `filter` | No | [Filter](#filter) | Provides the filter configuration.
-`start_time` | No  | String | The start of the time range during which to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword that represents the current `LocalDateTime`, such as `now`. To define a time range, use `start_time` with either `end_time` or `range`. 
-`end_time` | No | String  | The end of the time range during which to scan objects from all the buckets. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`, or it can be configured to the keyword that represents the current `LocalDateTime`, such as `now`. To define a time range, use `end_time` with either `start_time` or `range`. 
-`range` | No  | String  | The time range from which objects are scanned from all buckets. If configured with `start_time`, defines a last modified time range with `start_time` + `range`. If configured with `end_time`, defines a last modified time range with `end_time` - `range`. Supports ISO_8601 notation strings, such as `PT20.345S` or `PT15M`, and notation strings for seconds (`60s`) and milliseconds (`1600ms`). 
+`start_time` | No  | String | The time from which to start scanning the objects, all the objects modified after `start_time` will be scanned. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`. If `end_time` is configured along with `start_time`, all objects after `start_time` and before `end_time` will be processed. `start_time` and `range` cannot be used together. This will overwrite the `start_time` at [scan](#scan) level.
```

Comment:
```suggestion
`start_time` | No  | String | The time from which to start scanning objects modified after the given `start_time`. This should follow [ISO LocalDateTime](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) format, for example, `023-01-23T10:00:00`. If `end_time` is configured along with `start_time`, all objects after `start_time` and before `end_time` will be processed. `start_time` and `range` cannot be used together. This will overwrites the `start_time` at the [scan](#scan) level.
```